### PR TITLE
Fix queries, these have been validated as functional by client impl

### PR DESCRIPTION
**Pull request by prdlk**
I was running the blockchain with the client impl and realized that the query methods had a little glitch. This is a quick refactor and has been tested with HTTP, gRPC, and Dart Client.

https://user-images.githubusercontent.com/3804558/170404158-adbc545c-d396-4d74-ac09-cb2ca53c611c.mp4

 Mention [stepsize] in a comment if you'd like to report some technical debt. See examples here.

### CODE REVIEW

**Comment by prdlk** [comment]
@ntindle We would be ready to deploy v0.1.0 devnet after this is merged

**Review by ntindle — DISMISSED** [review]

File: x/registry/keeper/grpc_query_who_is.go [hunk 1]
```diff
@@ -47,11 +47,49 @@ func (k Keeper) WhoIs(c context.Context, req *types.QueryWhoIsRequest) (*types.Q
...
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	val, found := k.FindWhoIsByAlias(
+		ctx,
+		req.Alias,
+	)
+	if !found {
+		return nil, status.Error(codes.InvalidArgument, "not found")
```
Comment (mcjcloud) [inline x/registry/keeper/grpc_query_who_is.go]: `InvalidArgument` should be `NotFound`

File: x/registry/keeper/grpc_query_who_is.go [hunk 2]
```diff
@@ -47,11 +47,49 @@ func (k Keeper) WhoIs(c context.Context, req *types.QueryWhoIsRequest) (*types.Q
...
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	val, found := k.FindWhoIsByController(
+		ctx,
+		req.Controller,
+	)
+	if !found {
+		return nil, status.Error(codes.InvalidArgument, "not found")
```
Comment (mcjcloud) [inline x/registry/keeper/grpc_query_who_is.go]: Same as above, `NotFound`

File: x/registry/keeper/msg_server_who_is.go [hunk 3]
```diff
@@ -86,7 +86,7 @@ func (k msgServer) UpdateWhoIs(goCtx context.Context, msg *types.MsgUpdateWhoIs)
 			val.AddAlsoKnownAs(a, false)
 		}
 	}
-	val.Controllers = doc.ControllersAsString()
+	val.Controllers = doc.GetController()
```
Comment (mcjcloud) [inline x/registry/keeper/msg_server_who_is.go:89]: This method should be named `GetControllers`. I know it's not part of this PR but we should at least put an issue or just change it here.

**Review by mcjcloud — DISMISSED** [review]

**Comment by mcjcloud** [comment]
Why were the alias methods removed?

File: x/registry/keeper/msg_server_who_is.go (hunk 3 above)
Comment (prdlk) [reply x/registry/keeper/msg_server_who_is.go:89]: If anything WhoIs should be renamed to `Controller` that is the compliant property with the DID W3 Specification

**Comment by ntindle** [comment]
> Why were the alias methods removed?

Weren't the files just merged?

File: x/registry/keeper/msg_server_who_is.go (hunk 3 above)
Comment (joshLong145) [reply x/registry/keeper/msg_server_who_is.go:89]: This is out of scope.

**Review by joshLong145 — APPROVED** [review]

**Review by ntindle — APPROVED** [review]

**Pull request merged**